prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+<a href= "file:///Users/dadsmac/repos/prep-0621-code-solutions/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>;
```

Comment:
Here you'll want to use the directory and file names only for making a relative URL in your `href` attribute.

```html
href="contact-us/contact-us.html"
```

Otherwise, this link won't work properly unless someone else's file system matches yours exactly.

Reply:
https://user-images.githubusercontent.com/85319436/121185201-ac477080-c81a-11eb-9dae-796072613e95.mov